code/COVID19 US county Map-Daily Update.ipynb

```diff
@@ -2,33 +2,36 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 144,
+   "execution_count": 156,
    "metadata": {},
    "outputs": [],
    "source": [
     "import pandas as pd\n",
     "import numpy as np\n",
     "from datetime import datetime,timedelta\n",
     "import geopandas as gpd\n",
+    "from pathlib import Path\n",
     "pd.options.display.max_columns=100"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 208,
+   "execution_count": 157,
    "metadata": {},
    "outputs": [],
    "source": [
     "#data from github jhu,import the lastest data from timeseries\n",
     "df_Counties_confirmed=pd.read_csv(\"https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_confirmed_US.csv\")\n",
     "df_Counties_deaths=pd.read_csv(\"https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_time_series/time_series_covid19_deaths_US.csv\")\n",
-    "covid19Root = '/Users/adamlee/Documents/GitHub/COVID-19/data_tables/Data_for_UScounty_map'\n",
-    "covid19RootExport = '~/Documents/GitHub/COVID-19/data_tables/JHU_USCountymap_TEST/'"
+    "\n",
+    "#Root file variables\n",
+    "covid19Root = Path(\"~/Documents/GitHub/COVID-19/data_tables/Data_for_UScounty_map/\")\n",
+    "covid19RootExport = Path(\"~/Documents/GitHub/COVID-19/data_tables/JHU_USCountymap_TEST/\")\n"
```

Comment:
Consider building on the last definition, so if you've defined `covid19Root` you could define the other paths on top of it.  

Consider:

covid19Root = Path("~/Documents/GitHub/COVID-19/")
covid19_dataTables = covid19Root + ... path to data tables ...
covid19_export = covid19Root + ... export path ...

The reason this is also better is you can get covid19Root down to one variable you can load separately, outside of the script ( [environment variables ](https://medium.com/chingu/an-introduction-to-environment-variables-and-how-to-use-them-f602f66d15fa)) and all the other paths will build from that.